scripts/decrypt.sh

```diff
@@ -1,15 +1,23 @@
 #!/usr/bin/env bash
-# This script decrypts our test database 
-# WARNING: Edit decrypt.template.mustache not decrypt.sh which is a generated file
+# This script decrypts our test database
 # Bash3 Boilerplate. Copyright (c) 2014, kvz.io
+# You will need the environment variables CIRCLE_CI_KEY and CIRCLE_CI_IV
 
 set -o errexit
 set -o pipefail
 set -o nounset
 set -o xtrace
 
-if [[ "${TESTING_PROFILE}" =~ ^(confidential-workflow-tests|confidential-tool-tests|automated-review|singularity-tests)$ ]]; then
-    openssl aes-256-cbc -K $encrypted_292dafccc281_key -iv $encrypted_292dafccc281_iv -in secrets.tar.enc -out secrets.tar -d
-    tar xvf secrets.tar
-    mv dockstore-cli-integration-testing/src/test/resources/dstesting_pcks8.pem /home/travis/dstesting_pcks8.pem
-fi
+: "$CIRCLE_CI_KEY"
+: "$CIRCLE_CI_IV"
+
+openssl aes-256-cbc -d -in circle_ci_test_data.zip.enc -k "$CIRCLE_CI_KEY" -iv "$CIRCLE_CI_IV" -out secrets.tar
+tar xvf secrets.tar
+# Create the directory where the .pem file will be placed for continuous integration
+# If you would like to change this path (might be a good idea since we no longer use
+# TravisCI for CI) you need to change the path in dockstoreTest.yml which currently
+# is in the encrypted bundle circle_ci_test_data.zip.enc
+sudo mkdir -p /home/travis
```

Comment:
Shouldn't this be changed to w/e natalie used?

Reply:
Yes but the path is located in the encrypted bundle; there is no partialDockstoreTest.yml as in dockstore/dockstore; but maybe I'll just go ahead an fix it in the encrypted bundle.

Reply:
I fixed it